### ISSUE TITLE
Enable dynamic sitemap for this site setting won't save

### ISSUE THREAD

**Issue by mdavismcd** [ManOnDaMoon/omeka-s-module-Sitemaps#2]
When I check the box next to "Enable dynamic sitemap for this site," and click Save. I get the "Site successfully updated" green message, but when I go back to the site settings it's unchecked. I don't see anything in the log either, but maybe I'm not digging deep enough.

![Gif of the error](https://lh3.googleusercontent.com/u/0/drive-viewer/AFGJ81o4HnXLq7iOfzC0cSyWOE8WZv0yY-kfdanJXPgaxfzFNAc1CyGmq2NiqX35BnYx7fdW99hzpdEFI2NPjKp_BFHGO6gg=w2880-h1358)

[edit] Omeka S version 4.0.1 sitemaps version 1.0

**Comment by ManOnDaMoon** [comment]
Hello,

Would you be able to test the sitemap-index branch for the same issue? It's on its way to be released. I'll speed things up if you confirm it fixes your issue.

**Comment by mdavismcd** [comment]
For sure! I was able to save the setting with the -index branch. I see there are a couple other options as well. I'm not sure if this is the intended result, but here's a link to the site I'm trying to get the map for: https://digitalprojects.davidson.edu/omeka/s/college-archives-davidson-encyclopedia/sitemap.xml

I have more than 500 pages so I updated the results to 5000 and got three separate sitemaps. 

Thanks for all your work on this!
